app/views/spree/orders/_totals_footer.html.haml

```diff
@@ -23,6 +23,22 @@
           = t :order_total_price
       %td.text-right.total
         %h5#order_total= order.display_total.to_html
+    %tr.total
+      %td.text-right{colspan: "3"}
+        %h5
+          = t :order_amount_paid
+      %td.text-right.total
+        %h5
+          = order.payment_total
+  - if order.outstanding_balance?
+    %tr.total
+      %td.text-right{colspan: "3"}
+        %h5.not-paid
+          = t :order_balance_due
+      %td.text-right.total.not-paid
+        %h5.not-paid#balance_due
```

Comment:
I'm not sure if this comment was intended, but I think there's enough context not to need it:

```suggestion
        %h5.not-paid
```

Reply:
Removed comment.